File: app/src/main/java/com/dani_chuks/andeladeveloper/starwars/dagger/qualifiers/ComputationScheduler.java
package com.dani_chuks.andeladeveloper.starwars.dagger.qualifiers;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Qualifier;

@Qualifier
//@Target({ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER})
@Retention(RetentionPolicy.RUNTIME)
public @interface ComputationScheduler {

}

File: app/src/main/java/com/dani_chuks/andeladeveloper/starwars/dagger/qualifiers/ComputationScheduler.kt
package com.dani_chuks.andeladeveloper.starwars.dagger.qualifiers

import javax.inject.Qualifier

@Qualifier
//@Target({ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER})
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
annotation class ComputationScheduler
